### DISCUSSION TITLE
Nouvel outil pour mesurer son Ecoindex

### DISCUSSION THREAD

**Discussion by Fred-artwai** [cnumr/best-practices#400] [Show and tell]
Bonjour à toutes et à tous,

Ce message pour vous annoncer qu'artwaï met a disposition un simple formulaire pour tester son Ecoindex.
Cela se passe ici : https://ecoindex.artwai.com/
Bien sur, nous nous sommes assuré avec Frédéric Bordage que notre démarche était conforme aux conditionx d'utilisation décrit par le CNumR.

Evidemment, si il y a des couacs nous essaierons de les corriger rapidement.

Cordialement,
Frédéric Pineau

**Comment by Samiii-AP** [comment]
Bonjour,
Ca serait top si le site ecoindex et le vôtre n'attendaient pas le protocole "http / https" pour l'URL.
Je pense que c'est rédhibitoire pour certaines personnes qui ne comprennent pas l'erreur "Please enter a URL" puisqu'ils pensent l'avoir fait. Ce n'est pas explicit.
